mobile/src/main/java/mycroft/ai/activities/HomeMycroftAiActivity.java

```diff
@@ -0,0 +1,77 @@
+/*
+ *  Copyright (c) 2017. Mycroft AI, Inc.
+ *
+ *  This file is part of Mycroft-Android a client for Mycroft Core.
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package mycroft.ai.activities;
+
+import android.app.ProgressDialog;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.webkit.WebChromeClient;
+import android.webkit.WebSettings;
+import android.webkit.WebView;
+import android.webkit.WebViewClient;
+
+import mycroft.ai.R;
+
+public class HomeMycroftAiActivity extends AppCompatActivity {
+    private ProgressDialog progDailog;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_home_mycroft_ai);
+
+        progDailog = ProgressDialog.show(this, "Loading","Please wait...", true);
+        progDailog.setCancelable(false);
+
+        WebView webView = (WebView)findViewById(R.id.webview_home_mycroft_ai);
+
+        //webView.getSettings().setJavaScriptEnabled(true);
+        //webView.getSettings().setDomStorageEnabled(true);
+
+        webView.setWebViewClient(new WebViewClient() {
+            @Override
+            public boolean shouldOverrideUrlLoading(WebView view, String url) {
```

Comment:
Please use the non-deprecated method (with a `WebResourceRequest` parameter) instead.